daily_markov.py

```diff
@@ -0,0 +1,81 @@
+
+import praw
+import requests
+import time
+import markovify
+from unidecode import unidecode
+import re
+import random
+
+from bot import *
+
+FOOTER = '\n\n-----\n\n[^^Info](https://github.com/trambelus/UserSim) ^^| [^^Subreddit](/r/User_Simulator)'
```

Comment:
Let me know if you still want this moved to a bot.py I was hesitant to do it as if someone else were to fork this for their sub it's not actually something that should change. It's specifically for excluding the /u/user_simulator bot's footer from markov chains and for potential inclusion in the output quote for attribution purposes.